src/transformers/pipelines/__init__.py

```diff
@@ -256,7 +257,7 @@
     "image-segmentation": {
         "impl": ImageSegmentationPipeline,
         "tf": (),
-        "pt": (AutoModelForImageSegmentation,) if is_torch_available() else (),
+        "pt": (AutoModelForImageSegmentation, AutoModelForSemanticSegmentation) if is_torch_available() else (),
         "default": {"model": {"pt": "facebook/detr-resnet-50-panoptic"}},
```

Comment:
If we only have a single pipeline for all tasks, we can also only have one default model :/

i.e. it's not possible to showcase semantic segmentation with a default one

Reply:
Defaults are always limited :(.

But I think users tend to mostly copy paste blogposts + doc examples so if we use `pipeline(model="nvidia/segformer-b5-finetuned-ade-640-640")` in those, it should help people discover them, wdyt ?

Reply:
This is actually a good remark, that makes me think about pipelines vs tasks, and make we could imagine providing aliases `semantic-segmentation` that resolves to another default ? Food for thought.